recipe/emailverification/recipeimplementation/recipeimplementation.go

```diff
@@ -16,61 +20,62 @@ func NewRecipeImplementation(querier supertokens.Querier) *RecipeImplementation
 	}
 }
 
-func (r *RecipeImplementation) createEmailVerificationToken(userId, email string) map[string]interface{} {
+func (r *RecipeImplementation) createEmailVerificationToken(userId, email string) schema.ReturnMap {
```

Comment:
I don't think that using `schema.ReturnMap` solves the issue that I had talked about earlier.. Did you try and use `enum`?